src/main/python/vyu/timer.py

```diff
@@ -0,0 +1,24 @@
+import time
+
+
+class Timer(object):
+
+    def __init__(self):
+        self.t0 = None
+
+    def start(self):
+        self.t0 = time.time()
+
+    def clear(self):
+        self.t0 = None
+
+    def isrunning(self):
+        return self.t0 is not None
+
+    def check_time(self, dt):
```

Comment:
I wasn't sure what "dt" stood for.
(It's delta time)

Reply:
changed with commit 3af6389481a025457c35f0e03661c72f93d15880